### PR TITLE
[devtools] What's New in DevTools (Chrome 89)

### PR DESCRIPTION
**Pull request by chybie**
**Target Live Date:** YYYY-MM-DD

- [x] This has been reviewed and approved by (@chybie )
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Review by mathiasbynens — APPROVED** [review]

**Review by mathiasbynens — COMMENTED** [review]
Rubberstamp LGTM. I did notice the images are fairly large — some of them over 1 MB! Are they optimized with Squoosh/ImageOptim? (We could also create AVIF versions but it probably doesn't make sense to do that until after migration to the new site.)

**Comment by chybie** [comment]
> Rubberstamp LGTM. I did notice the images are fairly large — some of them over 1 MB! Are they optimized with Squoosh/ImageOptim? (We could also create AVIF versions but it probably doesn't make sense to do that until after migration to the new site.)

Erm.. I imageoptim all. Let me check again. Thanks for notice that.

File: src/content/en/updates/2021/01/devtools.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
...
+You can now set breakpoints and catch exceptions on Trusted Type Violations in the **Sources** panel.
+
+[Trusted Types](https://github.com/w3c/webappsec-trusted-types) API helps you prevent DOM-based cross-site scripting vulnerabilities. Learn how to write, review and maintain applications free of DOM XSS vulnerabilities with Trusted Types [here](https://web.dev/trusted-types/).
+
+In the **Sources** panel, open the **debugger** sidebar pane. Expand the **CSP Violation Breakpoints** section and enable the **Trusted Type violations** checkbox to pause on the exceptions. Try it yourself with [this demo page](https://tt-enforced.glitch.me/).
+
+![Breakpoint on Trusted Type violations](/web/updates/images/2021/01/devtools/trusted-type-violations.png)
+
+Could you add a picture explaining that the new issue icon at the end of the line shows issues raised at that line? Clicking the icon should take the user to the issues tab.
+
```
Comment (oslego) [inline src/content/en/updates/2021/01/devtools.md:30]: This is probably a leftover comment from document review.

Comment (chybie) [reply src/content/en/updates/2021/01/devtools.md:30]: thanks for notify, updated!

**Pull request merged**